pyconcz/settings/base.py

```diff
@@ -178,32 +178,28 @@
 
 SLACK_WEBHOOK = ''  # This variable is set in local.py
 
-TALKS_DATES = [datetime.date(2018, 6, 1), datetime.date(2018, 6, 2)]
-WORKSHOPS_DATES = [datetime.date(2018, 6, 3)]
+TALKS_DATES = [datetime.date(2019, 6, 14), datetime.date(2019, 6, 15)]
+WORKSHOPS_DATES = [datetime.date(2019, 6, 16)]
 TALKS_ROOMS = [
-    (1, 'Main'),
-    (2, 'Theatre'),
+    (1, 'Ball room'),
+    (2, 'Club'),
 ]
 WORKSHOPS_ROOMS = [
-    (4, 'room 301'),
-    (5, 'room 302'),
-    (6, 'room 303'),
-    (7, 'room 343'),
-    (8, 'room 346'),
-    (9, 'room 347'),
-    (10, 'room 348'),
-    (11, 'room 349'),
-    (20, 'Respirium'),
+    (4, 'room EB126'),
```

Comment:
I think we can omit "room "

Reply:
Of course, we can, but I'd leave it as is to make sure that everybody understands what these numbers mean.